modifiers/string.go

```diff
@@ -89,6 +90,28 @@ func SnakeCase(ctx context.Context, t *mold.Transformer, v reflect.Value, param
 	return nil
 }
 
+// AlphaCase removes non-alpha unicode characters. Example: "!@£$%^&'()Hello 1234567890 World+[];\" -> "HelloWorld"
+// From https://github.com/leebenson/conform
+func AlphaCase(ctx context.Context, t *mold.Transformer, v reflect.Value, param string) error {
+	s, ok := v.Interface().(string)
+	if !ok {
+		return nil
+	}
+	v.SetString(regexp.MustCompile(`[^\pL]`).ReplaceAllLiteralString(s, ""))
+	return nil
+}
+
+// NotAlphaCase removes alpha unicode characters. Example: "Everything's here but the letters!" -> "' !"
+// From https://github.com/leebenson/conform
+func NotAlphaCase(ctx context.Context, t *mold.Transformer, v reflect.Value, param string) error {
+	s, ok := v.Interface().(string)
+	if !ok {
+		return nil
+	}
+	v.SetString(regexp.MustCompile(`[\pL]`).ReplaceAllLiteralString(s, ""))
```

Comment:
Can we make the regexes global for reuse

Reply:
I agree. Done in other PR.